python/kfserving/kfserving/storage.py

```diff
@@ -193,9 +193,10 @@ def _download_local(uri, out_dir=None):
     @staticmethod
     def _create_minio_client():
         # Remove possible http scheme for Minio
-        url = re.compile(r"https?://")
-        minioClient = Minio(url.sub("", os.getenv("S3_ENDPOINT", "")),
+        url = urlparse(os.getenv("AWS_ENDPOINT_URL", ""))
```

Comment:
looks `urlparse` is not imported according to lint report.

Reply:
hi @yuzisun, sorry about that, I was betrayed by my IDE :) anyway, this is fixed now but I still get errors that seem unrelated?